NetUtils.py

```diff
@@ -397,12 +397,12 @@ def get_missing(self, state: typing.Dict[typing.Tuple[int, int], typing.Set[int]
                 location_id not in checked]
 
     def get_remaining(self, state: typing.Dict[typing.Tuple[int, int], typing.Set[int]], team: int, slot: int
-                      ) -> typing.List[int]:
+                      ) -> typing.List[typing.Tuple[int, int]]:
         checked = state[team, slot]
         player_locations = self[slot]
-        return sorted([player_locations[location_id][0] for
-                       location_id in player_locations if
-                       location_id not in checked])
+        return sorted([(player_locations[location_id][1], player_locations[location_id][0]) for
```

Comment:
It looks like this is taking these 2 elements that were in one order in one place, and swapping them for a different order in this new place.
Is there a reason behind the order you chose?
Were there any discrepancies in the order of these elements in other places before this PR?

Reply:
It's clearer in _speedups.pyx; these two elements are "receiver" (1) and "item id" (0), respectively. It's done this way to keep items sorted by the game they're for, because otherwise multiple games with item ids in the same ranges will result in their items getting mixed together.